test/utils.py

```diff
@@ -121,6 +122,9 @@ class TestContainerFile:
     default_stream_index: int
     stream_infos: Dict[int, Union[TestVideoStreamInfo, TestAudioStreamInfo]]
     frames: Dict[int, Dict[int, TestFrameInfo]]
+    custom_frame_mappings_data: Dict[
```

Comment:
Nit, since we should only access the mappins through the `get_custom_frame_mappings()` function, I'd suggest to "mark" that field as private:

```suggestion
    _custom_frame_mappings_data: Dict[
```